sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/cep/PatternCondition.java

```diff
@@ -26,6 +26,7 @@
  */
 public abstract class PatternCondition implements Serializable {
 
+  @SuppressWarnings("unused") // Affects a public api
```

Comment:
Is it possible to delete this variable and instead put the annotation on the constructor?

Reply:
Yes, changed